lib/tasks/templates.js

```diff
@@ -101,7 +101,7 @@ module.exports = function(gulp, config) {
 
 		templatesData.devmode = config.devmode;
 		templatesData.production = !config.devmode;
-		templatesData.buildstamp = config.devmode ? '' : config.stamp
+		templatesData.buildstamp = (config.buildstamp && !config.devmode) ? config.stamp : ''
```

Comment:
Hlavní je tahle změna. Původní podmínka byla špatně, protože v `jade` byla vždy vyplněná proměnná `buildstamp` bez ohledu na to, jestli vůbec build task něco dělá. Nešlo tedy v `mango.local.yaml` přepsat buildstamp task na `[]`/`false` a tím task úplně vypnout.